.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  Build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v3.0.0
+        with:
+          go-version: 1.17
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-
+
+      - name: Build
+        run: make
+
+      - name: Release
```

Comment:
Is there a title or description we can set about the release?

Reply:
Do you mean title/description to the [step name](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsname) in the GitHub workflow, or the resulting GitHub release? For the former only `name` is available; for the latter the name of the release is (currently) set to the git hash and the tag to `tip`. See an example [here](https://github.com/zregvart/ec-cli/releases/tag/tip).

Reply:
I was thinking the release. Just something to describe it's a dev build or if there's anything we can provide that's informative. It's not a big deal.